app/models/monologue/tag.rb

```diff
@@ -7,7 +7,18 @@ def posts_with_tag
     self.posts.published
   end
 
+  def self.with_frequency
+    self
+      .select("name, COUNT(*) as frequency")
+      .joins("INNER JOIN monologue_taggings ON monologue_taggings.tag_id = monologue_tags.id")
+      .joins("INNER JOIN monologue_posts    ON monologue_posts.id        = monologue_taggings.post_id")
```

Comment:
Line is too long. [103/80]<br>Place the . on the previous line, together with the method call receiver.